src/components/general/Header.js

```diff
@@ -6,10 +6,10 @@ const Header = () => (
   <Container>
     <img src="https://climateconnect.earth/logo.png"></img>
     <LinkContainer>
-      <Link href="forum" text="Forum" passHref></Link>
-      <Link href="browse" text="Browse" passHref></Link>
-      <Link href="create" text="Create A Project" passHref></Link>
-      <Link href="signin" text="SIGN IN" passHref type="button"></Link>
+      <Link href="forum" text="Forum"></Link>
+      <Link href="browse" text="Browse"></Link>
+      <Link href="create" text="Create A Project"></Link>
+      <Link href="signin" text="SIGN IN" type="button"></Link>
```

Comment:
Nitpick: this capitalization is stylistic, not semantic. Could we do one of the following?

1. Add `text-transform: uppercase` or similar
1. Add a TODO/issue to do that?

Reply:
Good call, did option 1 for `type="button"`